src/tests/Microsoft.PowerFx.Core.Tests/FunctionInfoTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public class FunctionInfoTests
+    {
+        [Fact]
+        public void FunctionInfo()
+        {
+            // Get any function info. 
+            var engine = new Engine();
+            var infos = engine.FunctionInfos.ToArray();
+            var infoAbs = infos.Where(info => info.Name == "Mid").First();
```

Comment:
>infos.Where(info => info.Name == "Mid").First(); [](http://example.com/codeflow?start=26&length=48)

Shorter version: infos.Single(info => info.Name == "Mid")

Reply:
I actually tried this and it's somehow returning 2 identical copies! That's weird, will investigate separately.